### PR TITLE
RM-927 Change operator tag source

### PR DESCRIPTION
**Pull request by cap1984**

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | failed |
| scaling | failed |
| recreate | failed |
| affinity | failed |
| demand-backup | failed |

commit: https://github.com/percona/percona-postgresql-operator/pull/57/commits/47be55c9d71ea64dcc7e003ff5cac2b5b931d5d5
image: `perconalab/percona-postgresql-operator:PR-57-47be55c9-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-57-47be55c9-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-57-47be55c9-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-57-47be55c9-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-57-47be55c9-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-57-47be55c9-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | failed |
| scaling | failed |
| recreate | failed |
| affinity | failed |
| demand-backup | failed |

commit: https://github.com/percona/percona-postgresql-operator/pull/57/commits/c1a09a987039c1675ab824e24a0dd6c7c7512ab7
image: `perconalab/percona-postgresql-operator:PR-57-c1a09a98-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-57-c1a09a98-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-57-c1a09a98-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-57-c1a09a98-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-57-c1a09a98-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-57-c1a09a98-pgo-deployer`

File: e2e-tests/functions [hunk 1]
```diff
@@ -80,7 +80,7 @@ deploy_operator() {
 	yq r -d'2' ${src_dir}/deploy/operator.yaml 'data[values.yaml]' \
 		| $sed -e "s#namespace: .*#namespace: ${namespace}#g" \
 		| $sed -e "s#pgo_operator_namespace: .*#pgo_operator_namespace: ${namespace}#g" \
-		| $sed -e "s#pgo_image_tag: .*#pgo_image_tag: ${GIT_BRANCH}#g" \
+		| $sed -e "s#pgo_image_tag: .*#pgo_image_tag: $(echo ${IMAGE_OPERATOR%-operator-postgres} | cut -d: -f2)#g" \
```
Comment (hors) [inline e2e-tests/functions:83]:
```suggestion
		| $sed -e "s#pgo_image_tag: .*#pgo_image_tag: $(echo ${IMAGE_OPERATOR%-postgres-operator} | cut -d: -f2)#g" \
```
postgres-operator ?

**Review by hors — CHANGES_REQUESTED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | failed |
| scaling | failed |
| recreate | failed |
| affinity | failed |
| demand-backup | failed |

commit: https://github.com/percona/percona-postgresql-operator/pull/57/commits/1c303d143ea75a4b910845b5cd1bc74552a73e42
image: `perconalab/percona-postgresql-operator:PR-57-1c303d14-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-57-1c303d14-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-57-1c303d14-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-57-1c303d14-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-57-1c303d14-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-57-1c303d14-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | failed |
| scaling | failed |
| recreate | failed |
| affinity | failed |
| demand-backup | failed |

commit: https://github.com/percona/percona-postgresql-operator/pull/57/commits/68609c6bd5b6d2c5050129e565ef4d53f79cf0fb
image: `perconalab/percona-postgresql-operator:PR-57-68609c6b-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-57-68609c6b-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-57-68609c6b-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-57-68609c6b-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-57-68609c6b-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-57-68609c6b-pgo-deployer`

File: e2e-tests/functions (hunk 1 above)
Comment (cap1984) [reply e2e-tests/functions]: Fixed
worked well in https://cloud.cd.percona.com/view/PGO/job/pgo-operator-gke-version/16/

**Pull request merged**